pkgs/by-name/sc/scalpel/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, autoconf
+, automake
+, libtool
+, tre
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "scalpel";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "sleuthkit";
+    repo = "scalpel";
+    rev = "35e1367";
+    hash = "sha256-0lqU1CAcWXNw9WFa29BXla1mvABlzWV+hcozZyfR0oE=";
+  };
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    libtool
+    tre
+  ];
+
+  postPatch = ''
+    sed -i \
+      -e 's|#define\s*SCALPEL_DEFAULT_CONFIG_FILE\s.*"scalpel.conf"|#define SCALPEL_DEFAULT_CONFIG_FILE "${placeholder "out"}/share/scalpel/scalpel.conf"|' \
```

Comment:
Just to know:
This config file can be overridden?

Reply:
Yes, basically there is a default user configuration that you can override. And you need to specify a default one.

Reply:
This is fine.

Maybe the only concern my paranoid side would point out is if this default config file is not referring to dangerous paths, or if any syntax errors on this file would prevent the whole program to run.